### PR TITLE
Base image updates

### PR DESCRIPTION
**Pull request by BenTheElder**
- newest containerd 1.3 which has some fixes
- initial base image cross build
ref: #188 #166 #1347 ...

this is _much_ slower and problematic to locally test (the manifest list cannot be exported to local, so you need to do a quick build and test that) but, it is feasible. starting with just amd64 and arm64.

### CODE REVIEW

File: pkg/build/node/node.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ import (
 const DefaultImage = "kindest/node:latest"
 
 // DefaultBaseImage is the default base image used
-const DefaultBaseImage = "kindest/base:v20200221-f3727da0@sha256:a9da2114e7ea293d9ac544a0c5b58d63e5ee44b4e76891bf35671fb1f5ffc611"
+const DefaultBaseImage = "kindest/base:v20200226-6d5ec59c@sha256:7fbe0890dab2dfb170fef3bf8ca8dda9bae40c66e6124a59ce2fd2debd18e818"
```
Comment (BenTheElder) [inline pkg/build/node/node.go]: you cannot browse the digest on docker hub, but the tag is browsable, so we include both.
this is the digest of a manifest list for linux/amd64 + linux/arm64

File: images/base/Makefile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+TAG?=$(shell echo "$$(date +v%Y%m%d)-$$(git describe --always --dirty)")
+IMAGE?=kindest/base:$(TAG)
+
+# build with buildx
+PLATFORMS?=linux/amd64,linux/arm64
```
Comment (BenTheElder) [inline images/base/Makefile]: we'll probably want to expand this, however I'm trying to wrap this up and make sure the base image works at all, my main priority was the containerd update, this change is more of a diversion due to #1347 #166

**Comment by aojea** [comment]
Cool

**Comment by BenTheElder** [comment]
FYI @neolit123

**Comment by neolit123** [comment]
ack, thanks for the heads up.

**Comment by BenTheElder** [comment]
shouldn't affect anything I think, BUT it is a manifest list image instead now, and it does mean some movement towards arm being a thing.

I'm guessing kinder users are all amd64 but I wouldn't be too surprised if someone was trying to use another architecture :sweat_smile:

**Comment by BenTheElder** [comment]
(and the image build is changing, we really need to remove `kind build base-image` which is outdated and have people leverage the makefile / not need to build it themselves since we'll have pre-cross-built-it for them)

**Comment by neolit123** [comment]
we have quite a few amd64 jobs for kinder right now and the amd64 assumptions are hardcoded.

i'm hoping that one day we will be able to run at least one arm64 job too.

since we don't have a way to provision arm64 with prow today (AFAIK), i was wondering if its possible to try it in github actions, by [requesting a non-amd64 VM in the workflow job](https://github.com/uraimo/run-on-arch-action#optional-parameters) and running kind/kinder there with k8s non-amd64 artifacts.

but this has been in the backlog, due to other priorities.

**Comment by BenTheElder** [comment]
Ack, thanks for the details! I'll follow up on this in the future.

**Comment by BenTheElder** [comment]
Looks like that action is just a qemu wrapper, the performance will probably be too bad.

Travis has ACTUAL arm IIRC but getting it in k8s may be hard.

**Comment by neolit123** [comment]
> Looks like that action is just a qemu wrapper, the performance will probably be too bad.

i didn't know it is using emulation. this might be quite slow, indeed.

**Comment by BenTheElder** [comment]
locally using qemu to build the image is pretty slow, and I'm betting the actions workers are much smaller machines.

**Pull request merged**